corehq/apps/export/views.py

```diff
@@ -1251,7 +1345,7 @@ def fmt_export_data(self, export):
             'addedToBulk': False,
             'exportType': export.type,
             'emailedExports': emailed_exports,
-            'editUrl': reverse(EditCustomCaseExportView.urlname,
+            'editUrl': reverse(edit_view.urlname,
```

Comment:
What happens if have the toggle set but you click on an old export? Will this break?